clients/py/setup.py

```diff
@@ -25,40 +25,32 @@ def version():
                 _, version = line.split('=')
                 return version.replace("'", '').strip()
 
+def is_ignored(line):
+    line = line.strip()
+    return (not line) or (line[0] == "#")
 
-def load_deps(section):
-    """Load dependencies from Pipfile, we can't assume toml is installed"""
-    # [packages]
-    header = '[{}]'.format(section)
-    with open('Pipfile') as fp:
-        in_section = False
+def load_deps(path):
+    """Load dependencies from requirements file"""
+    with open(path) as fp:
+        deps = []
         for line in fp:
-            line = line.strip()
-            if not line:
-                continue
-
-            if line == header:
-                in_section = True
+            if is_ignored(line):
                 continue
+            line = line.strip()
 
-            if line.startswith('['):
-                in_section = False
+            # e.g.: git+https://github.com/nuclio/nuclio-jupyter.git@some-branch#egg=nuclio-jupyter
+            if "#egg=" in line:
+                _, package = line.split("#egg=")
+                deps.append(f"{package} @ {line}")
```

Comment:
git+https://github.com/nuclio/nuclio-jupyter.git@some-branch#egg=nuclio-jupyter @ nuclio-jupyter
Is this what you expect here?

Reply:
I see that you got your inspiration from here:
https://github.com/mlrun/mlrun/blob/2e8417125b964e3afa2af14567218b18cb67160a/setup.py#L44
So I guess it's ok... :)